api-extractor/src/DocItemLoader.ts

```diff
@@ -229,7 +229,13 @@ export default class DocItemLoader {
       }
     );
 
-    const packageName: string = path.basename(packageJsonFilePath).split('.').shift();
+    // Include the scope name in the cache key
+    // For out test cases, the will be no searching the alias path, 
+    // so we manually include the '@types' scope name
+    const scopeName: string = isTypes ?
+      '@types' : packageJsonFilePath.match(/@\w+/).toString() ;
```

Comment:
The motivation for this approach was that the package name and scope name are included in the path. If I take the route of reading the package.json file then we have to probe upwards until we find the package.json file then read the file... seems kind of slow when the information we need is already in the path?